config/locales/en.yml

```diff
@@ -318,11 +319,13 @@ en:
           paid: Paid
           confirmed: Confirmed
           canceled: Canceled
-          initiated: "Waiting PayPal payment"
```

Comment:
It would be great to create a new translation key instead of editing the existing one. We rely on open-source contributors for the translations and it is possible that, for some languages, they won't be updated quickly. In other languages, they could still display "Waiting PayPal payment" (in the translated language) for a while, even if the EN version isn't the initial one. To avoid this, it's better to create a new key: it might be that it isn't translated shortly but at least the English message that will be visible will be the correct one.